modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupLocalServiceImpl.java

```diff
@@ -39,10 +42,56 @@
 public class AccountGroupLocalServiceImpl
 	extends AccountGroupLocalServiceBaseImpl {
 
-	/**
-	 * NOTE FOR DEVELOPERS:
-	 *
-	 * Never reference this class directly. Use <code>com.liferay.account.service.AccountGroupLocalService</code> via injection or a <code>org.osgi.util.tracker.ServiceTracker</code> or use <code>com.liferay.account.service.AccountGroupLocalServiceUtil</code>.
-	 */
+	@Override
+	public AccountGroup addAccountGroup(
+			long userId, String name, String description)
+		throws PortalException {
+
+		long accountGroupId = counterLocalService.increment();
+
+		AccountGroup accountGroup = accountGroupPersistence.create(
+			accountGroupId);
+
+		User user = userLocalService.getUser(userId);
+
+		accountGroup.setCompanyId(user.getCompanyId());
+		accountGroup.setUserId(user.getUserId());
+		accountGroup.setUserName(user.getFullName());
+
+		accountGroup.setName(name);
+		accountGroup.setDescription(description);
+
+		return accountGroupPersistence.update(accountGroup);
+	}
+
+	@Override
+	public AccountGroup deleteAccountGroup(AccountGroup accountGroup) {
+		return super.deleteAccountGroup(accountGroup);
+	}
+
+	@Override
+	public AccountGroup deleteAccountGroup(long accountGroupId)
+		throws PortalException {
+
+		return deleteAccountGroup(getAccountGroup(accountGroupId));
+	}
+
+	@Override
+	public AccountGroup updateAccountGroup(
+			long accountGroupId, long userId, String name, String description)
+		throws PortalException {
+
+		AccountGroup accountGroup = accountGroupPersistence.fetchByPrimaryKey(
+			accountGroupId);
+
+		User user = userLocalService.getUser(userId);
+
+		accountGroup.setCompanyId(user.getCompanyId());
```

Comment:
@alee8888 @pei-jung This seems a bit odd to me. I think we only want to deal with setting the `companyId` during creation. Otherwise, it should never change. We can just remove the `userId` param from this method altogether.

Reply:
I removed it in 2e3116e38dbc5ee53c3213730d789cf0cf8aa76f

Reply:
👍 thanks @pei-jung!

Reply:
Sorry I missed that